config/sidekiq.yml

```diff
@@ -1,6 +1,7 @@
 ---
 :queues:
   - default
+  - mailers
```

Comment:
Just checking - did you mean to bump the mailers queue priority?

Reply:
I did. It just seemed like sending emails to users was more important than the stats snapshot. Probably doesn't matter as they both have a tiny amount of jobs on them for the most part anyway. They're both almost always empty.